rhobs/alerting/data_plane/prometheus.availability_metric_konflux_alerts.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-konflux-up-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: konflux_up_alerts
+    interval: 1m
+    rules:
+    - alert: KonfluxAlert
+      expr: count ((konflux_up offset 1h) unless (konflux_up))
+      for: 1m
+      labels:
+        severity: warning
+      annotations:
+        summary: Availability metric 'konflux_up' is missing.
+        description: >-
+          Availability metric 'konflux_up' is missing some entries when compared to
+          last one hour.
```

Comment:
We should include the relevant `konflux_up` labels. Debugging this alert would be really difficult without them.

Reply:
Since the count function is returning the value, it's not showing the labels. Now, I have removed the count and added labels.